app/models/queen.rb

```diff
@@ -1,2 +1,21 @@
 class Queen < Piece
+  # Can move horizontal, vertical, or diagonal for unlimited spaces unless path blocked
+  def valid_move?(x,y)
+
+    # Returns false if path is blocked to destiantion space
+      return false if path_blocked?(x,y)
```

Comment:
I think this is `is_obstructed?` in our game - I don't think we have a method called `path_blocked?`

Reply:
You are right, that was my fault. But same concept.